enclone_exec/tests/enclone_test3.rs

```diff
@@ -170,18 +170,23 @@ fn test_site_examples() {
 #[test]
 fn test_enclone_examples() {
     PrettyTrace::new().on();
-    for t in 0..EXAMPLES.len() {
-        let testn = format!("{}", EXAMPLES[t]);
-        let out_file = format!("../enclone_help/src/example{}", t + 1);
+    let examples = [
+        (1, r###"BCR=123089 CDR3=CARRYFGVVADAFDIW"###),
+        (
+            2,
+            r###"BCR=123085 GEX=123217 LVARSP=gex,IGHV2-5_g_μ CDR3=CALMGTYCSGDNCYSWFDPW"###,
+        ),
+    ];
+    for (num, args) in examples {
+        let out_file = format!("../enclone_help/src/example{}", num);
```

Comment:
We could make the variable inline in format strings
```suggestion
        let out_file = format!("../enclone_help/src/example{num}");
```

Reply:
Indeed, though this is a lint issue in a lot of places in this codebase, and this was a very mechanical refactor.  I want to get this in without waiting for a full CI cycle.  We can definitely clean this up en masse at some point.